lite-variables.tf

```diff
@@ -53,36 +53,29 @@ variable "aks_admin_ip_cidr_map" {
 variable "aks_node_sizes" {
   description = "The VM size for each node of each node pool in the HPCC Systems. Example format is '{ roxie = \"xlarge\", serv = \"2xlarge\", spray = \"xlarge\", thor = \"xlarge\" }'.\nSee https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-general for more information."
   type = object({
-    roxie           = optional(string, "2xlarge")
-    serv            = optional(string, "4xlarge")
-    spray           = optional(string, "2xlarge")
-    thor            = optional(string, "2xlarge")
+    roxie           = optional(string, "xlarge")
+    serv            = optional(string, "2xlarge")
+    spray           = optional(string, "large")
+    thor            = optional(string, "xlarge")
   })
-}
+  validation {
+      condition = (length(regexall("^[24]*x*large", var.aks_node_sizes.roxie)) == 1) && (length(regexall("^[24]*x*large", var.aks_node_sizes.serv)) == 1) && (length(regexall("^[24]*x*large", var.aks_node_sizes.spray)) == 1) && (length(regexall("^[24]*x*large", var.aks_node_sizes.thor)) == 1)
 
-variable "aks_capacity" {
-  description = "The min and max number of nodes of each node pool in the HPCC Systems. Example format is '{ roxie_min = 1, roxie_max = 3, serv_min = 1, serv_max = 3, spray_min = 1, spray_max = 3, thor_min = 1, thor_max = 3}'."
-  type = object({
-    roxie_min       = optional(number, 1)
-    roxie_max       = optional(number, 3)
-    serv_min        = optional(number, 1)
-    serv_max        = optional(number, 3)
-    spray_min       = optional(number, 3)
-    spray_max       = optional(number, 6)
-    thor_min        = optional(number, 3)
-    thor_max        = optional(number, 6)
-  })
+      error_message = "All aks_node_sizes must be one of the following: large, xlarge, 2xlarge, or 4xlarge."
+  }
   default  = {
-    roxie_min       = 1
-    roxie_max       = 3
-    serv_min        = 1
-    serv_max        = 3
-    spray_min       = 3
-    spray_max       = 6
-    thor_min        = 3
-    thor_max        = 6
+    roxie       = "xlarge"
+    serv        = "2xlarge"
+    spray       = "large"
+    thor        = "xlarge"
   }
 }
+
+variable "aks_thorpool_max_capacity" {
```

Comment:
The user should not be required to set the maximum number of nodes in a node pool.  The user should simply indicate the size of the thor they want (e.g. 200-way) and how many simultaneous jobs they want to run and we figure out the rest.